src/main/java/seedu/address/model/person/Appointment.java

```diff
@@ -122,8 +142,20 @@ public boolean equals(Object other) {
         }
 
         Appointment otherAppointment = (Appointment) other;
-        return otherAppointment.reason.equals(reason)
+
+        return otherAppointment.patient.getName().equals(patient.getName())
```

Comment:
```suggestion
        return otherAppointment.patient.equals(patient)
```
Is it possible to shorten this line to code to this? Would it break anything?